docs/source/tutorials/SAR/SAR tutorial.ipynb

```diff
@@ -0,0 +1,1180 @@
+    "    if is_sar:\n",
+    "        if syn_nosyn:\n",
+    "            env = SynNoSynWrapper(gym.make(env_name), ica, pca, scaler, phi)\n",
+    "        else:\n",
+    "            env = SynergyWrapper(gym.make(env_name), ica, pca, scaler)\n",
+    "    else:\n",
+    "        env = gym.make(env_name)\n",
+    "\n",
+    "\n",
+    "    camera = 'side_view'\n",
+    "        \n",
+    "    for i,__ in tqdm(enumerate(range(episodes))):              \n",
+    "        env.reset()\n",
+    "\n",
+    "        model = SAC.load(model_path)\n",
+    "        vec = VecNormalize.load(env_path, DummyVecEnv([lambda: env]))\n",
+    "        \n",
+    "        rs = 0\n",
+    "        is_solved = []\n",
+    "        done = False\n",
+    "        for _ in range(1000):\n",
+    "            o = vec.normalize_obs(env.get_obs())\n",
+    "            a, __ = model.predict(o, deterministic=determ)\n",
+    "            \n",
+    "            frame = env.sim.renderer.render_offscreen(width=640, height=480,camera_id=camera)\n",
+    "            frames.append(frame)\n",
+    "\n",
+    "            next_o, r, done, info = env.step(a)\n",
+    "            is_solved.append(info['solved'])\n",
+    "            \n",
+    "            rs+=r\n",
+    "    env.close()\n",
+    "    skvideo.io.vwrite(f'{video_name}.mp4', np.asarray(frames),outputdict={\"-pix_fmt\": \"yuv420p\"})\n",
+    "    \n",
+    "ica,pca,norm = load_locomotion_SAR()\n",
+    "env_path = 'multihill_terrain_syn_env_myoLegWalkHillyTerrain-v0_3'\n",
+    "model_path = 'multihill_terrain_syn_model_myoLegWalkHillyTerrain-v0_3.zip'\n",
+    "get_video('Test-v0',model_path,env_path,1,'test',pca=pca,ica=ica,scaler=norm,is_sar=True)\n",
+    "show_video('test.mp4')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc299ac0",
+   "metadata": {},
+   "source": [
+    "# SAR core functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c8433315",
+   "metadata": {},
+   "source": [
+    "<div style=\"display: flex; align-items: flex-start;\">\n",
+    "    <div style=\"flex: 1;\">\n",
+    "         Next, we'll define and implement the core functions for SAR. The framework follows the basic structure displayed on the right. Accordingly, we will define functions that do all of the following:\n",
+    "        <br>\n",
+    "        <br>\n",
+    "        <ol>\n",
+    "            <li>Train a policy on a simpler task</li>\n",
+    "            <li>Rollout and capture muscle activation data from this trained policy</li>\n",
+    "            <li>Compute SAR from this activation data</li>\n",
+    "            <li>Use SAR to train on a harder task (see policy architecture below)</li>\n",
+    "        </ol>\n",
+    "        <br>\n",
+    "        <br>\n",
+    "        We can now define functions to implement each of these steps.\n",
+    "    </div>\n",
+    "    <div>\n",
+    "        <img src=\"./SAR_images/framework.png\" alt=\"Framework\" style=\"width:200px;\"/>\n",
```

Comment:
Make the background transparent for the image.

I believe you are using the white color scheme. I use black and this image looks quite off with it. Removing the background will help a lot.

Reply:
updated this figure to make it much more explanatory and help with flow